File: java/seasar/annotation/src/test/DataImpl.java
package test;

import org.seasar.framework.container.annotation.tiger.*;

@Component(name="data")
public class DataImpl implements Data {

	private String name;
	private int point;

	public DataImpl() {
		this("test", 100);
	}

	public DataImpl(String name, int point) {
		this.name = name;
		this.point = point;
	}

	public String getName() {
		return this.name;
	}

	public int getPoint() {
		return this.point;
	}

}

File: csharp/AspectDNG+S2.NET/extend-s2container/DataImpl.cs

using System;

public class DataImpl : IData
{
	private string name;
	private int point;

	public DataImpl(string name)
	{
		this.name = name;
	}

	public int Point
	{
		set
		{
			this.point = value;
		}
	}

	public void Print()
	{
		Console.WriteLine("name={0}, point={1}", this.name, this.point);
	}
}
